package.json

```diff
@@ -75,5 +75,8 @@
   "bugs": {
     "url": "https://github.com/code-dot-org/ml-activities/issues"
   },
-  "homepage": "https://github.com/code-dot-org/ml-activities#readme"
+  "homepage": "https://github.com/code-dot-org/ml-activities#readme",
+  "dependencies": {
+    "react-typist": "^2.0.5"
```

Comment:
:grin: https://jstejada.github.io/react-typist/

Does this belong in `devDependencies` since we're bundling it?